### PR TITLE
[Feat] Home Page

### PR DESCRIPTION
**Pull request by rushikesh3299**
Home Page
Mobile responsive

image
image
image

### CODE REVIEW

File: css_files/navbar.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        display: none;
+    }
+
+    .nav-link{
+        display: none;
+    }
+
+    .nav-icon-link-login{
+        display: none;
+    }
```
Comment (amarphule) [inline css_files/navbar.css:85]: you could create single class and assign it to the element

File: css_files/navbar.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+.mobile-nav{
+    position: fixed;
+    width: 200px;
```
Comment (amarphule) [inline css_files/navbar.css:90]: you can use `rem`

**Review by amarphule — COMMENTED** [review]
Looks good to me, given some minor suggestion

File: css_files/homepage.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    display: flex;
+    gap: 3rem;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.featured-card{
+    padding: 1rem;
+    width: 250px;
+    height: 270px;
```
Comment (harshitadaswani) [inline css_files/homepage.css:102]: use `rems` to make it responsive

File: css_files/homepage.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+.features-brand-section{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 3rem;
+    margin: 3rem auto; 
+}
+
+.features-brand-section img{
+    width: 200px;
```
Comment (harshitadaswani) [inline css_files/homepage.css:129]: here too

**Review by harshitadaswani — APPROVED** [review]
use `rems to make images responsive

File: css_files/homepage.css (hunk 4 above)
Comment (rushikesh3299) [reply css_files/homepage.css]: Done. Thanks for suggestions

File: css_files/navbar.css (hunk 2 above)
Comment (rushikesh3299) [reply css_files/navbar.css]: Done. Thanks for suggestions

**Pull request merged**